tmail-backend/event-bus-postgres/src/main/java/org/apache/james/events/KeyChannelMessage.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.james.events;
+
+import org.apache.commons.lang3.StringUtils;
+
+public record KeyChannelMessage(EventBusId eventBusId, String routingKey, String eventAsJson) {
```

Comment:
I think that code seems very similar to existing KeyChannelMessage for redis pub sub and could be generified in case this work gets merged maybe too?

Reply:
Yes, indeed this class is generic.